UIViewPreviewProvider.swift

```diff
@@ -48,6 +48,10 @@ public protocol UIViewPreviewProvider {
     
     /// Array of `Preview` to be displayed in the Xcode canvas  pane `(⌥ + ⌘ + return)`
     static var uiPreviews: [Preview] { get }
+    
+    /// `ColorScheme` to be used in the Xcode canvas pane
+    @available(iOS 13.0, *)
+    static var colorScheme: ColorScheme { get }
```

Comment:
This means we'd need to create two different `UIViewPreviewProvider` in order to preview both light mode and dark mode.

Lets make this an array instead:
```swift
static var colorSchemes: [ColorScheme] { get }
```

which we can default to:
```swift
public static var colorSchemes: [ColorScheme] {
    [.light, .dark]
}
```